taxi/views.py

```diff
@@ -20,14 +30,14 @@ def index(request):
     num_visits = request.session.get("num_visits", 0)
     request.session["num_visits"] = num_visits + 1
 
-    context = {
+    contexte = {
         "num_drivers": num_drivers,
         "num_cars": num_cars,
         "num_manufacturers": num_manufacturers,
         "num_visits": num_visits + 1,
     }
 
-    return render(request, "taxi/index.html", context=context)
+    return render(request, "taxi/index.html", context=contexte)
```

Comment:
There is a typo in the variable name `contexte`. It should be `context` to match the parameter name in the `render` function call.